pkg/operator/ceph/object/spec.go

```diff
@@ -280,17 +280,34 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 }
 
 func (c *clusterConfig) createCaBundleUpdateInitContainer(rgwConfig *rgwConfig) v1.Container {
-	caBundleMount := v1.VolumeMount{Name: caBundleVolumeName, MountPath: caBundleSourceCustomDir, ReadOnly: true}
+	caBundleMount := v1.VolumeMount{Name: caBundleVolumeName, MountPath: "/tmp/ca-bundle", ReadOnly: true}
 	volumeMounts := append(controller.DaemonVolumeMounts(c.DataPathMap, rgwConfig.ResourceName, c.clusterSpec.DataDirHostPath), caBundleMount)
 	updatedCaBundleDir := "/tmp/new-ca-bundle/"
 	updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: updatedCaBundleDir, ReadOnly: false}
 	volumeMounts = append(volumeMounts, updatedBundleMount)
 	return v1.Container{
 		Name:    "update-ca-bundle-initcontainer",
 		Command: []string{"/bin/bash", "-c"},
```

Comment:
Because this is getting more complex, let's use xtrace so users can debug potential issues with what the script is doing in their environments.

Let's also use errexit so we can be sure the container (and pod) will fail if commands like `cut`/`tr`/`cat` aren't present in someone's build.

```suggestion
		Command: []string{"/bin/bash", "-c", "-x", "-e"},
```

Reply:
added